internal/lexer/gen/golang/lexer.go

```diff
@@ -180,39 +180,39 @@ func (this *Lexer) Scan() (tok *token.Token) {
 
 			switch rune1 {
 			case '\n':
-				this.line++
-				this.column = 1
+				l.line++
+				l.column = 1
 			case '\r':
-				this.column = 1
+				l.column = 1
 			case '\t':
-				this.column += 4
+				l.column += 4
```

Comment:
I still dislike this set amount of 4 spaces for a tab and would still prefer it counting as 1, removing this whole case. I should probably try to do something about #35.